website/content/en/docs/reference/configuration/_index.md

```diff
@@ -310,8 +310,6 @@ data_dir = "/var/lib/vector"
 [api]
 enabled = false
 # address = "127.0.0.1:8686"
-
-'''
```

Comment:
This just junk?

Reply:
Yeah, stray copy-paste I think. [link](https://vector.dev/docs/reference/configuration/#automatic-namespacing)

Reply:
Similarly, https://github.com/vectordotdev/vector/blob/master/config/examples/namespacing/vector.toml is a tested version of this and doesn't have it.